samples/agent-catalog/msft-agent-samples/foundry-agent-service-sdk/browser-automation-agent/README.md

```diff
@@ -51,15 +51,18 @@ The agent is configured through a Python script (`browser_automation.py`) and au
       - AI Services resource
       - AI Project
       - Model deployment
-3. Playwright Resource setup: 
-   - Create a Microsoft Playwright Resource: https://learn.microsoft.com/en-us/azure/playwright-testing/how-to-manage-playwright-workspace?tabs=playwright
-   - Generate an API Key for the Playwright resource: https://learn.microsoft.com/en-us/azure/playwright-testing/how-to-manage-access-tokens
+3. Playwright Workspace Resource setup: 
+   - Create a Playwright Workspace Resource: https://aka.ms/pww/docs/manage-workspaces
+   - Generate an Access Token for the Playwright Workspace resource: https://aka.ms/pww/docs/manage-access-tokens
    - Access the Workspace region endpoint from the Workspace Details. 
-4. Create a serverless connection in the Azure AI Foundry project with the Playwright workspace region endpoint and the Playwright Access Key. 
+   - Give the Project Identity a "Contributor" role on the Playwright Workspace resource, or configure a custom role by following these instructions: https://aka.ms/pww/docs/manage-workspace-access
+4. Create a serverless connection in the Azure AI Foundry project with the Playwright workspace region endpoint and the Playwright Workspace Access Token. 
    - Go to the Azure AI Foundry portal and select your AI Project. Go to the Management center and Click connected resources.
    - Create a new connection of type Serverless Model and enter the following information.
-      - Target URI - Playwright Workspace Region Endpoint (example - wss://eastus.api.playwright.microsoft.com/accounts/eastus_xxxxxxxxxxxxxxxxxxxxxxx/browsers).
-      - Key - Playwright Access Key
+      - Target URI - Playwright Workspace Region Endpoint (example - wss://eastus.api.playwright.microsoft.com/playwrightworkspaces/eastus_xxxxxxxxxxxxxxxxxxxxxxx/browsers).
```

Comment:
new id format doesnt have region_ so pls remove eastus_ and we could paramterized region as well so make it this wss://{region}.api.playwright.microsoft.com/playwrightworkspaces/{workspaceId}/browsers